src/collection.js

```diff
@@ -494,6 +494,21 @@ class Collection {
     return this
   }
 
+  /**
+   * Returns median of the current collection
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  async median () {
+    const arr = (await this.sort()).items

```

Comment:
I’d prefer to just await the sorting without reassigning the sorted values to `arr` here:

```suggestion
    await this.sort()
```

Reply:
Done. Please Check